src/Uno.UI/Microsoft/UI/Xaml/Controls/SplitButton/SplitButton.cs

```diff
@@ -209,7 +209,7 @@ private void OnVisualPropertyChanged(DependencyObject sender, DependencyProperty
 		internal void UpdateVisualStates(bool useTransitions = true)
 		{
 			// place the secondary button
-			if (m_lastPointerDeviceType == PointerDeviceType.Touch || m_isKeyDown)
+			if (m_isKeyDown)
```

Comment:
Is this change coming from WinUI?

Reply:
@MartinZikmund yes. This is available in in the latest 1.7 preview of winappsdk.

Issue for ref: https://github.com/microsoft/microsoft-ui-xaml/issues/178

Based on Microsoft changes: microsoft/microsoft-ui-xaml@6909712#diff-26435cd7d5a1b50dca1fba8a84c5432c4941c8be555ba8d4ff8e34d99987fb9a

SplitButton.cs
https://github.com/microsoft/microsoft-ui-xaml/blob/69097129a853c65a16447aade4c82576d4724b1a/src/controls/dev/SplitButton/SplitButton.cpp#L121

SplitButtonTests_InteractionTests.cs
https://github.com/microsoft/microsoft-ui-xaml/blob/69097129a853c65a16447aade4c82576d4724b1a/src/controls/dev/SplitButton/InteractionTests/SplitButtonTests.cs#L110-L144